src/coreclr/src/jit/compiler.cpp

```diff
@@ -469,49 +474,6 @@ var_types Compiler::getJitGCType(BYTE gcType)
     return result;
 }
 
-#ifdef ARM_SOFTFP
```

Comment:
I guess this appeared to be statically dead code?

Reply:
Yes, this was only callable when FEATURE_HFA was true, except that ARM_SOFTFP also disables FEATURE_HFA.

---
In reply to: [510494655](https://github.com/dotnet/runtime/pull/43706#discussion_r510494655) [](ancestors = 510494655)